CellProfiler64.iss

```diff
@@ -45,7 +45,6 @@ Name: "{userappdata}\CellProfiler\ijplugins"; Flags: uninsneveruninstall
 Source: ".\dist\CellProfiler.exe"; DestDir: "{app}"; Flags: ignoreversion
 Source: ".\dist\analysis_worker.exe"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
Yes we could - it's a little bit of legacy at this point. I think it makes sense to standardize, but we have to take out some code in cellprofiler.analysis that expects analysis_worker.exe to be there in the frozen version of CellProfiler.